### PR TITLE
FAQ update

### PR DESCRIPTION
**Pull request by NateKomodo**
Added elite related FAQ and changed the tense for game end date

### CODE REVIEW

File: bot/cogs/cyber.py [hunk 1]
```diff
@@ -143,8 +143,8 @@ def contains_words(self, text: str, all_words: Iterable[str], *any_words: Iterab
         text = (message.content).lower()
 
         # CyberStart Game Dates.
-        if self.contains_words(text, ["when", "game"], ["does", "will"], ["end", "finish", "close"]):
-            await message.channel.send(f"{message.author.mention}  |  Cyberstart Game ends on the 29th May.")
+        if self.contains_words(text, ["when", "game"], ["does", "will", "did"], ["end", "finish", "close"]):
+            await message.channel.send(f"{message.author.mention}  |  Cyberstart Game ended on the 29th May.")
 
         # CyberStart Essentials Dates.
         elif self.contains_words(text, ["when", "essentials"], ["?", "does", "will"], ["end", "finish", "close"]):
```
Comment (thebeanogamer) [inline bot/cogs/cyber.py]: This bit needs updating too.

File: bot/cogs/cyber.py [hunk 2]
```diff
@@ -155,6 +155,16 @@ def contains_words(self, text: str, all_words: Iterable[str], *any_words: Iterab
             text = f"{message.author.mention}  |  **Quote from the @CyberDiscUK Twitter:**"
             text += "Selection for CyberStart Elite will be based on a combination of Game and Essentials results."
             await message.channel.send(text)
+            
+        # CyberStart Elite Dates.
+        elif self.contains_words(text, ["when", "elite"], ["?", "does", "will"], ["start", "begin", "end", "run"]):
+            await message.channel.send(f"{message.author.mention}  |  Cyberstart Elite dates: London - 4th and 5th August, Bristol - 28th and 29th July, Manchester - 21st and 22nd July")    
+        
+        # CyberStart Elite email.
+        elif self.contains_words(text, ["havent", "haven't", "elite", "email", "got"]):
```
Comment (thebeanogamer) [inline bot/cogs/cyber.py]: This might get spammed a lot, unsure if it's appropriate. Will consult others.

File: bot/cogs/cyber.py [hunk 3]
```diff
@@ -155,6 +155,16 @@ def contains_words(self, text: str, all_words: Iterable[str], *any_words: Iterab
             text = f"{message.author.mention}  |  **Quote from the @CyberDiscUK Twitter:**"
             text += "Selection for CyberStart Elite will be based on a combination of Game and Essentials results."
             await message.channel.send(text)
+            
+        # CyberStart Elite Dates.
+        elif self.contains_words(text, ["when", "elite"], ["?", "does", "will"], ["start", "begin", "end", "run"]):
```
Comment (thebeanogamer) [inline bot/cogs/cyber.py]: This is good and helpful.

File: bot/cogs/cyber.py [hunk 4]
```diff
@@ -143,8 +143,8 @@ def contains_words(self, text: str, all_words: Iterable[str], *any_words: Iterab
         text = (message.content).lower()
 
         # CyberStart Game Dates.
-        if self.contains_words(text, ["when", "game"], ["does", "will"], ["end", "finish", "close"]):
-            await message.channel.send(f"{message.author.mention}  |  Cyberstart Game ends on the 29th May.")
+        if self.contains_words(text, ["when", "game"], ["does", "will", "did"], ["end", "finish", "close"]):
+            await message.channel.send(f"{message.author.mention}  |  Cyberstart Game ended on the 29th May.")
```
Comment (thebeanogamer) [inline bot/cogs/cyber.py]: Well done for noticing. This is good.

**Comment by thebeanogamer** [comment]
I will disable the thing that checks if the line is too long though.

**Comment by NateKomodo** [comment]
Fixed issues and made a tweak to the strings and the criteria for the elite email FAQ

**Pull request merged**